.github/workflows/ci.yml

```diff
@@ -6,50 +6,80 @@ on:
 
 env:
   BUILD_TYPE: Release
+  WORKING_DIRECTORY: ${{ github.workspace }}/build
```

Comment:
not sure we need that var. It is not a working dir in every step so its name may be confusing. Maybe it should be `BUILD_DIR`?
But from docs I also see that they don't prefix it with `${{github.workspace}}` so it is pretty much `BUILD_DIR: ./build`. With such a simple value I'm not convinced it actually brings much value.

Reply:
I introduced this global variable as this path is used in multiple places. This is only for consistency and to not repeat.

I agree that the name is not the best. Do you want me to revert this change or only change the variable's name?